Makefile

```diff
@@ -24,14 +24,14 @@ SRC = src/debug/main_a.c\
 		src/builtins/ft_export.c\
 		src/builtins/ft_unset.c\
 		src/builtins/ft_exit.c\
-		src/debug/print.c\
```

Comment:
Interestingly, we had this line in Makefile twice (line 16 too). gcc didn't care, cc complained, and it took me ages to figure out why :D